### DISCUSSION TITLE
Output

### DISCUSSION THREAD

**Discussion by Nicat-dcw** [community/community#139190] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Your response for all of models is not pretty enough can u use similar openai? Ex: {
  "id": "cmpl-82b68197b48f4a66b4ff27e9de599175",
  "object": "chat.completion",
  "created": 1726836041,
  "model": "Meta-Llama-3.1-70B-Instruct",
  "choices": [
    {
      "index": 0,
      "message": {
        "role": "assistant",
        "content": "The capital of France is Paris.",
        "tool_calls": []
      },
      "finish_reason": "stop"
    }
  ],
  "usage": {
    "prompt_tokens": 28,
    "completion_tokens": 8,
    "total_tokens": 36
  }
} And even can change model name like instead Meta-Llama... Change Llama-3.1-70b-instruct

**Comment by Sofines** [comment]
you can specify the desired format you're looking for and the structured output language (Markdown, JSON,..) or consider building a pipeline for fine-tuning the responses after generation
